include/mbedtls/cmac.h

```diff
@@ -151,12 +158,15 @@ int mbedtls_cipher_cmac_reset( mbedtls_cipher_context_t *ctx );
  *                      output = generic CMAC(cmac key, input buffer).
  *
  *
- * \param cipher_info   The cipher information.
- * \param key           The CMAC key.
+ * \param cipher_info   The cipher information. This must not be \c NULL.
+ * \param key           The CMAC key. This must not be \c NULL.
  * \param keylen        The length of the CMAC key in bits.
- * \param input         The buffer holding the input data.
+ * \param input         The buffer holding the input data. This must be a
+ *                      readable buffer of length \p ilen Bytes. It may be
+ *                      \c NULL if ilen == 0.
  * \param ilen          The length of the input data.
  * \param output        The buffer for the generic CMAC result.
```

Comment:
We must document the required size of `output` here. To my understanding, it's the block-size of the underlying cipher.